udata_front/tests/views/test_dataset.py

```diff
@@ -153,14 +152,6 @@ def test_200_if_deleted_but_authorized(self):
         response = self.get(url_for('datasets.show', dataset=dataset))
         self.assert200(response)
 
-    def test_no_index_on_empty(self):
```

Comment:
Maybe, we could rebrand it to `test_no_index_on_archived`? With an archived dataset as example?

Reply:
Done in 4fc7cf61